docs/resources/architect_schedules.md

```diff
@@ -0,0 +1,34 @@
+---
+page_title: "genesyscloud_architect_schedules Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud Architect Schedules
+---
+# genesyscloud_architect_schedules (Resource)
+
+Genesys Cloud Architect Schedules
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+**No APIs**
```

Comment:
This should pick up the APIs from your apis.md file when you run `go generate` it probably didn't pick it up because you misspelled genesys in `examples/resources/genensyscloud_architect_schedules/apis.md`

Reply:
Seems like the same misspelling with examples/resources/genensyscloud_architect_schedules/resource.tf as well :)